src/Microsoft.PowerShell.Commands.Utility/commands/utility/DebugRunspaceCommand.cs

```diff
@@ -101,19 +101,11 @@ public Guid InstanceId
         }
```

Comment:
@KirkMunro, your last commit had 1 failures in `PowerShell-CI-linux`
Default enablement of Experimental Features.On preview builds, Experimental Features are enabled
```powershell
Expected: Feature PSCommandNotFoundSuggestion to be Enabled
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/ExperimentalFeature/Get-ExperimentalFeature.Tests.ps1: line 176
176:             $expFeature.Enabled | Should -BeEnabled -Name $expFeature.Name
```